server/quota/node.go

```diff
@@ -0,0 +1,57 @@
+package quota
+
+import (
+	"context"
+
+	"github.com/tigrisdata/tigris/server/config"
+	"golang.org/x/time/rate"
+)
+
+// This limiter protects the node from overloading,
+// It limits the total number of request and throughput
+// handled by the node, by all the namespaces.
+
+type node struct {
+	state *State
+	cfg   *config.QuotaConfig
+}
+
+func (c *node) Allow(_ context.Context, namespace string, size int, isWrite bool) (err error) {
```

Comment:
not seeing much benefit of named return value here, same applies to below method.